_docs/backenders/strategy-2021/roles/hiring.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Hiring role definition -- *draft*
+
+**Current responsible**: -
+
+## Responsibilities
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+  
+    - Identified as responsibility to be shared with Capacity Role 
+
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles 
+    
+    
+- Distributing tasks: reviewing candidates, attending interviews
+
+
+- Liaising with P&C about onboarding of new BE members
+  
+    - Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+    
+    
+- Collaborating with P&C on job ads text
+  
+    - Identified as responsibility to be shared with Capacity, Strategy roles
+    
+    
+- Preparing candidate profile descriptions for outsourcing needs
+
+   - Identified as responsibility to be shared with Capacity Role
+
+
```

Comment:
```suggestion
- Preparing candidate profile descriptions for outsourcing needs

  *Responsibility to be shared with Capacity role*
```

Reply:
I think in practice these profiles will mostly be better handled by project developers with input from PMs, but I don't see any harm in keeping this here (especially if nobody is taking this role right now 😁)